repos/fdbt-site/src/components/DistanceRow.tsx

```diff
@@ -0,0 +1,259 @@
+                                    km
+                                </div>
+                            </div>
+                        </>
+                    </FormGroupWrapper>
+                </div>
+                <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                    {index === 0 ? (
+                        <FormGroupWrapper
+                            errors={errors}
+                            errorIds={[`cap-pricing-per-distance-minimum-price-${index}`]}
+                            hideErrorBar
+                        >
+                            <>
+                                <>
+                                    <label className="govuk-label" htmlFor="minimum-price">
+                                        Minimum price
+                                    </label>
+
+                                    <span className="govuk-hint" id={`minimum-price-hint-${index}`}>
+                                        e.g. 2.99
+                                    </span>
+                                </>
+                                <div className="govuk-input__wrapper">
+                                    <div className="govuk-input__prefix" aria-hidden="true">
+                                        £
+                                    </div>
+                                    <input
+                                        className="govuk-input govuk-input--width-5"
+                                        id={`minimum-price-${index}`}
+                                        name={`minimumPrice${index}`}
+                                        type="text"
+                                        spellCheck="false"
+                                        onChange={(e) => {
+                                            setCapPricingPerDistanceData({
+                                                ...capPricingPerDistanceData,
+                                                [`minimumPrice${index}`]: e.target.value,
+                                            });
+                                        }}
+                                        value={capPricingPerDistanceData[`minimumPrice${index}`] || ''}
+                                    />
+                                </div>
+                            </>
+                        </FormGroupWrapper>
+                    ) : (
+                        <div className="govuk-!-margin-right-2">
+                            <FormGroupWrapper
+                                errors={errors}
+                                errorIds={[`cap-pricing-per-distance-price-${index}`]}
+                                hideErrorBar
+                            >
+                                <>
+                                    <label className="govuk-label" htmlFor={`price-per-km-${index}`}>
+                                        {index === 1 ? 'Price per km' : 'Price'}
```

Comment:
I think the designs are just wrong here - it needs to say Price per km on every line. Don't always take the design too literally, always question why we are doing something different.